.github/scripts/common/lib.sh

```diff
@@ -539,3 +539,18 @@ parse_branch_names_from_backport_labels() {
   BRANCHES=$(echo "$BRANCHES" | sed 's/^ *//')
   echo "$BRANCHES"
 }
+
+# Extract the PR number from the PR title
+#
+# input: PR_TITLE
+# output: PR_NUMBER or exit 1 if the PR title does not contain the PR number
+extract_pr_number_from_pr_title() {
+  PR_TITLE=$1
+  if [[ "$PR_TITLE" =~ \#([0-9]+) ]]; then
```

Comment:
Your script is written with a `#!/bin/sh` shebang, but you're using Bash-specific syntax `[[ ... =~ ... ]]` and `BASH_REMATCH`, which is not supported by `/bin/sh` on many systems

Reply:
False alarm, I saw that you source this file with `. ./.github/scripts/common/lib.sh`, so the shebang won’t have any effect. But I still recommend considering an update to the shebangs.

Reply:
Yeah, I was thinking about it, will do it in the follow-up PR, as it will need extended testing, cause this script used in different CI flows